### ISSUE TITLE
EventFlow not working with .NET 5

### ISSUE THREAD

**Issue by ChrisRobbers** [Azure/diagnostics-eventflow#377]
Our app has been using EventFlow with the Microsoft.Extensions.Logging input and Azure EventHub output for many months, but when I tried upgrading the app to .NET 5 I noticed it stopped sending to our EventHub. Changing it back to .NET Core 3.1 caused it to start working again. I don't know if this is something specific to our configuration or something in general with EventFlow. Has this been tested against .NET 5 in general?

Here is our eventFlowConfig.json:

```
{
  "inputs": [
    {
      "type": "Microsoft.Extensions.Logging"
    }
  ],
  "outputs": [
    {
      "type": "EventHub",
      "eventHubName": "",
      "connectionString": "",
      "filters": [
        {
          "type": "drop",
          "include": "Level == Verbose"
        }
      ]
    }
  ],
  "settings": {
    "maxEventBatchSize": "100",
    "maxBatchDelayMsec": "5000"
  },
  "schemaVersion": "2016-08-11"
}
```

**Comment by karolz-ms** [comment]
@ChrisRobbers we are just getting to it. Thanks for the report, we'll look ASAP

**Comment by yantang-msft** [comment]
@ChrisRobbers Thanks for reporting this issue. I reproduced it and found this is because of this bug: https://github.com/Azure/azure-sdk-for-net/issues/13899
We will update our dependency on EventHub to the latest version (from 1.0.1 to 4.3.1, yes it's quite old) and this issue will go away.

**Comment by karolz-ms** [comment]
Many thanks @yantang-msft for investigation and the fix!

@ChrisRobbers a new version of the EventHub output with the fix (1.7.0) has been published on nuget.org. Please let us know if there are any issues with it, otherwise enjoy ☺️
